includes/compat/class-wc-stripe-email-admin-failed-refund.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * An email sent to the admin when a refund fails.
+ *
+ * @since 9.6.0
+ */
+class WC_Stripe_Email_Admin_Failed_Refund extends WC_Stripe_Email_Failed_Refund {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->id          = 'wc_stripe_failed_refund_admin';
+		$this->description = __( 'Refund request failure emails are sent to chosen recipient(s) when an attempt to process refund fails.', 'woocommerce-gateway-stripe' );
```

Comment:
Nit: We can simplify to "Refund failure", and I think we should add an `a` to the description.
```suggestion
		$this->description = __( 'Refund failure emails are sent to chosen recipient(s) when an attempt to process a refund fails.', 'woocommerce-gateway-stripe' );
```